tests/test.py

```diff
@@ -1001,6 +1001,27 @@ def random_keys(size):
                      '--keys_public_output_dir={}'.format(folder)],
                     env=random_keys(key_size * 2))
 
+    def test_gen_keys_with_empty_client_id(self):
+        #keys not needed client_id for generation
+        with tempfile.TemporaryDirectory() as folder:
+            subprocess.check_output(
+                ['./acra-keymaker', '--keystore={}'.format(KEYSTORE_VERSION),
+                 '--keys_output_dir={}'.format(folder),
+                 '--client_id=""',
+                 '--generate_poisonrecord_keys',
+                 '--generate_log_key',
+                 '--generate_acrawebconfig_keys',
+                 '--keys_public_output_dir={}'.format(folder)])
+
+        with tempfile.TemporaryDirectory() as folder:
+            with self.assertRaises(subprocess.CalledProcessError) as exc:
+                subprocess.check_output(
+                    ['./acra-keymaker', '--keystore={}'.format(KEYSTORE_VERSION),
+                     '--keys_output_dir={}'.format(folder),
+                     '--client_id=''',
```

Comment:
what will print next code:
```
a =  '--client_id='''
print(a)
```
?

I suggest to rewrite this literal to `'--client_id'` or `"--client_id=''"` to be more readable what the actual value will be used. Because now we check `client_id` validation, not empty value)

Reply:
If you provide just `--client_id` it wont be parsed at all and default value will be used. 
So agree to use `"--client_id=''"`

Reply:
Actually, this code will produce just `-client_id=` which is totally fine for our scenario, but in case of simply passing `--client_id` it wont start because you are passing param but not specified its value:)

```
a =  '--client_id='''
print(a)
```